### DISCUSSION TITLE
Figma file availability for Dash-UI PRO

### DISCUSSION THREAD

**Discussion by IVANKRUPSKI** [codescandy/codescandy#24] [Q&A] [answered]
Sorry for writing here. You don't respond on my messages on your web site when I use "Contact us" form.

**Question:**
Could you please advise if the full-fledged (Dash-UI pro) Figma file is available for Dash-UI pro?
Can we obtain to test UX/UI before buying the full template?

Thank you!

**Comment by codescandy** [comment, marked as answer]
> Sorry for writing here. You don't respond on my messages on your web site when I use "Contact us" form.
> 
> **Question:** Could you please advise if the full-fledged (Dash-UI pro) Figma file is available for Dash-UI pro? Can we obtain to test UX/UI before buying the full template?
> 
> Thank you!

Hello @IVANKRUPSKI ,
We reply your email in the morning. can you please check the mail.
After purchase you can find the Figma File in the Readme.MD 
If you dont get let me know. We share you. 
Also, in the email we share you a Figma PRo File too.

Thank you let me know
